### ISSUE TITLE
Help wanted

### ISSUE THREAD

**Issue by sajanv88** [7leads/ngx-cookie-service#52]
## Expected Behavior
I want to read cookies from different domain

## Actual Behavior
... N/A

## Steps to Reproduce the Problem
... N/A

## Specifications

  - Version: ...
  - Browser: ...
  - ...

**Comment by sajanv88** [comment]
@CunningFatalist  I have read that already. There is no option to read cookies from another domain by using this library.

**Comment by CunningFatalist** [comment]
@sajanv88 Yes, that is why I linked the FAQ. For security reasons, what you are trying to do is impossible. This is not something we decided to do, it is browser default behaviour.
